components/transactions/TransactionItem.js

```diff
@@ -154,6 +156,36 @@ const TransactionItem = ({ displayActions, collective, transaction, onMutationSu
 
   const displayedAmount = getDisplayedAmount(transaction, collective);
 
+  const transactionDetailLink = transactionKind => {
```

Comment:
`transactionDetailsLink` as you use plural elsewhere.

Reply:
Done. 👍🏽